### PR TITLE
[MM-51475] - Monthly Subscriptions: Notifications to move to annual

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR changes the copy on the banners shown to cloud pro monthly subscribers and communicates the date we're ending support for cloud pro monthly

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-51475

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/admin_console/billing/billing_subscriptions/to_yearly_nudge_banner.test.tsx [hunk 1]
```diff
@@ -2,11 +2,9 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
+import {screen} from '@testing-library/react';
```
Comment (neallred) [inline webapp/channels/src/components/admin_console/billing/billing_subscriptions/to_yearly_nudge_banner.test.tsx:5]: :tada:

File: webapp/channels/src/components/admin_console/billing/billing_subscriptions/to_yearly_nudge_banner.tsx [hunk 2]
```diff
@@ -22,29 +23,112 @@ import {GlobalState} from '@mattermost/types/store';
...
+    Zero = '0'
+}
+
+// eslint-disable-next-line no-process-env
+const cloudProMonthlyCloseMoment = process.env.CLOUD_PRO_MONTHLY_DEPRECATION_DATE || '20230416';
+
+interface ToYearlyPlanDismissPreference {
+
+    // range represents the range for the days to the deprecation of cloud free e.g. in 30 to 10 days to deprecate cloud free
+    // Incase of dismissing the banner, range represents the time (days) period when this banner was dismissed.
```
Comment (neallred) [inline webapp/channels/src/components/admin_console/billing/billing_subscriptions/to_yearly_nudge_banner.tsx:41]: Nit `Incase` -> `In case`

**Review by neallred — DISMISSED** [review]
LGTM, thanks @AGMETEOR !

**Review by ConorMacpherson — DISMISSED** [review]

**Comment by AGMETEOR** [comment]
Hey @ConorMacpherson and @neallred , reviews went stale

**Review by neallred — APPROVED** [review]

**Review by neallred — APPROVED** [review]

**Review by ConorMacpherson — APPROVED** [review]

**Pull request merged**